xmtp/Cargo.toml

```diff
@@ -15,9 +15,10 @@ xmtp_networking = { path = "../xmtp_networking" }
 async-trait = "0.1.68"
 xmtp_cryptography = { path = "../xmtp_cryptography"}
 hex = "0.4.3"
-
-[dev-dependencies]
 ethers = "2.0.4"
```

Comment:
This will go back to a dev dependency once we have some way of passing in an external signer.